adapter/validator/src/main/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidator.java

```diff
@@ -95,39 +99,117 @@ public GtfsTypeValidator(@NotNull GtfsSpecificationProto.CsvSpecProto fileSchema
                     case INPUT_TYPE_UNSPECIFIED: // Text is default and does not require validation
                     case TEXT:
                     case COLOR:
-                    case DATE: //no special validation
+                    case DATE: // no special validation
                         break;
+                    case INTEGER:{
```

Comment:
Not defined in https://gtfs.mobilitydata.org/specifications/static-gtfs#h.hc443y62gb8c

Reply:
Will remove, since `NOT_NULL_INTEGER` AND `NON_NEGATIVE_INTEGER` cover our needs